narwhal/executor/src/state.rs

```diff
@@ -18,10 +18,10 @@ pub struct ExecutionIndices {
 }
 
 impl ExecutionIndices {
-    pub fn last_for_certificate(certificate_index: u64) -> Self {
+    pub fn last_for_commit(commit_round: u64) -> Self {
```

Comment:
nit: I think `end_for_commit` may be clearer, since this key is unlikely to exist in the `consensus_message_order` table.

Reply:
Makes sense!